File: C#/FC4.cs
using System;

namespace FC4
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter a number.");
            long n = Convert.ToInt64(Console.ReadLine());
            Console.WriteLine();
            for (long x = 1; x <= Math.Floor(Math.Sqrt(n)); x++)
                if (n % x == 0)
                    Console.WriteLine(n / x + ", " + x);
            Console.WriteLine();
            Console.WriteLine("List complete.");
        }
    }
}

File: Pascal/FC4.pas
program FC4;
Uses math;

var
n : Integer;
i : Integer;

begin
  writeln ('Enter a number:');
  readln (n);
  for i:=1 to Floor (Sqrt (n)) do
      if (n mod i)=0 then
         writeln (Floor(n/i), ' ', i);
  readln
end.
